### PR TITLE
npm-{run, start, stop, restart, test}: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

https://github.com/tldr-pages/tldr/issues/14000

### CODE REVIEW

**Comment by Managor** [comment]
It felt a bit silly to create stub pages for the reserved keywords, but it was the best way I could think of to document them.

File: pages/common/npm-restart.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# npm restart
+
+> Run a script named `restart`.
+> See also: `npm-run`.
+> More information: <https://www.npmjs.com>.
+
+- Run a script named `restart`:
+
+`npm run restart`
```
Comment (acuteenvy) [inline pages/common/npm-restart.md:9]: > It felt a bit silly to create stub pages for the reserved keywords, but it was the best way I could think of to document them.

Maybe alias pages would be better?

```suggestion
# npm restart

> This command is an alias of `npm run restart`.

- View documentation for the original command:

`tldr npm run`
```

Comment (Managor) [reply pages/common/npm-restart.md:9]: I like it.

File: pages/common/npm-run.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# npm run
+
+> Run a script.
+> More information: <https://www.npmjs.com>.
+
+- Run a script named `string`:
+
+`npm run {{string}}`
```
Comment (spageektti) [inline pages/common/npm-run.md:8]:
```suggestion
- Run a script:

`npm run {{script_name}}`
```

File: pages/common/npm-run.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Pass arguments to a script:
+
+`npm run {{string}} -- {{argument}} {{--option}}`
```
Comment (spageektti) [inline pages/common/npm-run.md:12]:
```suggestion
`npm run {{script_name}} -- {{argument}} {{--option}}`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**